### ISSUE TITLE
different train and val splits from the paper?

### ISSUE THREAD

**Issue by imyangs** [GibranBenitez/IPN-hand#5]
Hi @GibranBenitez 

Firstly thank you for publishing the dataset and code. 

I ran the isolated test successfully with your dataset and pretrained weights, but I got a different confusion matrix from the paper.  So I am wondering if I missed sth, or just you published different splits?
![1](https://user-images.githubusercontent.com/28534716/102585044-98ac2e80-4142-11eb-8c3a-714479664b39.JPG)

Moreover, when I tried to load resnet50 with pretrained weights, the shapes do not match. Please tell me how to fix it. Thx. 
![2](https://user-images.githubusercontent.com/28534716/102585229-fc365c00-4142-11eb-8d92-24256615c7b2.JPG)

**Comment by GibranBenitez** [comment]
Hi @imyangs,

Thank you for your interest in our work.
For the first point, just to confirm, are you using the RGB-F model?
As far as I remember, the splits published and those from the paper must be the same. However, please show the normalized results of the CM for a direct comparison.

By a quick check on the ResNet models, I am afraid that I forgot to change the number of classes from the jester dataset. Could you please check if it runs by using 25 instead of 13 classes for the main model?

**Comment by imyangs** [comment]
Thank @GibranBenitez for quick reply.

In my offline tests, the mean Acc with ResNeXt-101model using different modalities are very close to the paper:
RGB-F: 86.326% (86.32%）
RGB-S: 84.686% (84.77%)
RGB   :  84.594% (83.59%)

But as u can see in my attached fig, the total number of video fragments for Val is 1097 that is different from the paper (1097 vs 1101). More specifically, the number of class Point-1f (label 1) and Point-2f are both over 200, which are around 100 in the reported CM. 
![4](https://user-images.githubusercontent.com/28534716/102732054-97fade80-4374-11eb-9014-a6a9484fa569.JPG)

By the way, I was using the annotation file from ./annotation_ipnGesture/ipnall_but_None.json. The number of validation items is 1097. 
![5](https://user-images.githubusercontent.com/28534716/102732451-b90fff00-4375-11eb-97a5-71e4b43b0503.JPG)

**Comment by imyangs** [comment]
For the ResNet-50, I ran the offline demo successfully when I changed the num of classes from 13 to 25, and it gave me reasonable results. RGB-F is 74.567%, RGB-S is 75.114%. 

So as u worried, maybe u forgot to change the num of classes.  @GibranBenitez
